tests/tests_unit/test_data_classes/test_data_models/test_typed_instances.py

```diff
@@ -231,6 +233,60 @@ def test_dump_load_person(self) -> None:
         assert isinstance(loaded.birth_date, date)
         assert all(isinstance(sibling, DirectRelationReference) for sibling in loaded.siblings or [])
 
+    @pytest.mark.dsl
+    def test_to_pandas(self) -> None:
+        import pandas as pd
+
+        person = PersonRead(
+            "sp_my_fixed_space", "my_external_id", 1, 0, 0, "John Doe", date(1990, 1, 1), "john@doe.com"
+        )
+
+        df = person.to_pandas()
+
+        pd.testing.assert_series_equal(
+            df,
+            pd.Series(
```

Comment:
This should not be a DataFrame

Reply:
I was a bit confused by that, no problem to change to dataframe